embedded-graphics/src/primitives/mod.rs

```diff
@@ -50,3 +50,13 @@ pub trait ContainsPoint {
     /// Returns `true` if the given point is inside the shape.
     fn contains(&self, point: Point) -> bool;
 }
+
+/// Offset outline trait.
+pub trait OffsetOutline {
+    /// Offsets the outline of the shape.
+    ///
+    /// The offset is applied perpendicular to each element of the outline.
+    /// Offset values larger than zero will expand the shape and values smaller
+    /// than zero will shrink the shape.
```

Comment:
Should we mention here that the behaviour is to saturate if the numbers are too large to fit in an `i32` or `u32`? It's unlikely to happen so might not be worth adding in.

Reply:
IMO it's not worth mentioning, because it's so unlikely for normal display sizes.